src/lambdas/api/app.js

```diff
@@ -129,17 +129,10 @@ app.get('/collections/:collectionId/items', async (req, res, next) => {
 })
 
 app.post('/collections/:collectionId/items', async (req, res, next) => {
-  try {
-    res.json(await api.searchItems(
-      req.params.collectionId,
-      req.body,
-      satlib.es,
-      req.endpoint,
-      'POST'
-    ))
-  } catch (error) {
-    next(error)
-  }
+  if (!process.env['ENABLE_TRANSACTIONS_EXTENSION']) next(createError(404))
```

Comment:
```suggestion
  if (!process.env['ENABLE_TRANSACTIONS_EXTENSION']) return next(createError(404))
```

As written, it's going to call the `next` callback twice.